mypy/typeshed/stdlib/@python2/__builtin__.pyi

```diff
@@ -121,7 +121,7 @@ class type(object):
     def __new__(cls, name: str, bases: Tuple[type, ...], namespace: Dict[str, Any]) -> type: ...
     def __call__(self, *args: Any, **kwds: Any) -> Any: ...
     def __subclasses__(self: _TT) -> List[_TT]: ...
-    # Note: the documentation doesnt specify what the return type is, the standard
+    # Note: the documentation doesn't specify what the return type is, the standard
```

Comment:
Could you make these changes in the separate typeshed repo instead? Mypy just copies it in.

Reply:
Sure, what's the repo address?

Reply:
github.com/python/typeshed, thanks! Please remove the typeshed changes from this PR, we'll carry them in the next time we sync typeshed into this repo.

Reply:
Reverted in  40e1fd8